src/components/dashboard/KeyPerformanceIndicator.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <div class="kpi-container mx-3">
+    <div class="centered">
+      <span class="has-text-weight-bold is-size-1 has-text-centered">
+        {{ value }}
+      </span>
+      <p class="has-text-centered has-text-weight-bold">
+        {{ title }}
+      </p>
+    </div>
+  </div>
+</template>
+
+<script lang="ts" setup>
+defineProps<{
+  title: string;
+  value: number | string;
+}>();
+</script>
+
+<style scoped>
+.kpi-container {
+  max-width: 200px;
```

Comment:
should probably be in `rem` instead of `px`

Reply:
Updated to `20rem`